src/ctest.ts

```diff
@@ -105,7 +105,7 @@ interface CTestInfo {
         backtrace: number;
         command: string[];
         name: string;
-        properties: { name: string; value: string }[];
+        properties: { name: string; value: string | string[] }[];
```

Comment:
Couldn't we simplify the logic farther down by simply making this just `string[]`? Then later we don't need the if condition.

Reply:
It can be a string though?

Reply:
String[] doesn't include string

Reply:
Right, but wouldn't it make it simpler if this is just an array and if there is only one string it's just a single element array? 

Or is this user facing and we're trying to make this easier for the user to enter a simple string?

Reply:
This is output from "CTest --show-only=json-v1". User don't need to change it directly.

Reply:
Gotcha, if it's not user facing then I would say we could just have `string[]`, but if you think it's better to have `string | string[]`, that's good with me!

Approved.

Reply:
We don't really have a choice since we are just parsing the output of CTest, and that field can be either a string or an array of strings.

Reply:
Makes sense to me. Thanks for the clarification!